build/configs/artik053/.flashSpec.xml

```diff
@@ -22,6 +22,7 @@
     <options>
         <option desc="Flash all binaries and images">ALL</option>
         <option desc="Flash TizenRT binary">OS</option>
+	<option desc="Automatic generate of usb rule">USBRULE</option>
```

Comment:
1. alignment
2. "Register the USB rule"

Reply:
modified